assets/app/views/_labels.html

```diff
@@ -0,0 +1,41 @@
+<div ng-controller="LabelsController" class="labels">
+<div ng-show="expanded">
+  <h3>Labels</h3>
+  <a href ng-click="toggleExpanded()">Collapse</a> 
+  <span class="help"><a href data-toggle="tooltip" data-placement="bottom" title="Labels are a way to filter objects you create in OpenShift.">Help</a> About Labels</span>
+  <div class="labels-edit">
+    <div class="edit-label">
+      <input type="text" placeholder="Label" ng-model="labelKey">
+      <span class="separator">:</span>
+      <input type="text" placeholder="Value" ng-model="labelValue">
+      <button ng-click="addLabel()">Add Label</button>
+    </div>
+    
+    <div ng-show="labels.Name">
+      <span class="key">Name</span>
+      <span>:</span>
+      <span class="value">{{ labels.Name }}</span>
+    </div>
+        
+    <div ng-repeat="(key,value) in labels" ng-if="key!='Name'">
+      <span class="key">{{ key }}</span>
+      <span>:</span>
+      <span class="value">{{ value }}</span>
+      <a href ng-click="deleteLabel(key)">Delete</a>
+    </div>
+  </div>
+</div>
+<div ng-hide="expanded">
+  <h3>Labels</h3> 
+  <a href ng-click="toggleExpanded()">Expand</a>
+  <span class="help"><a href data-toggle="tooltip" data-placement="bottom" title="Labels are a way to filter objects you create in OpenShift.">Help</a> About Labels</span>
```

Comment:
Yup, wasn't sure what was the best way to show it... I think I can move the common stuff above.